docs/livemedia-creator.rst

```diff
@@ -12,9 +12,10 @@ isos, bootable (partitioned) disk images, tarfiles, and filesystem images for
 use with virtualization and container solutions like libvirt, docker, and
 OpenStack.
 
-The general idea is to use qemu with kickstart and an Anaconda boot.iso to
-install into a disk image and then use the disk image to create the bootable
-media.
+livemedia-creator first uses a bootable ISO image containing Anaconda to start a
+virtual machine with qemu. Then, the virtual machine runs Anaconda with a Kickstart file
+you provide to install a Fedora system into a filesystem image. Last, the filesystem image
+is used by lorax to create the bootable media.
```

Comment:
lorax isn't used. some parts of pylorax and similar templates are, but I think 'by lorax' could just be dropped.